sharding/src/main/scala/com/lightbend/akka/bench/sharding/latency/LatencyBenchEntity.scala

```diff
@@ -71,17 +74,19 @@ object LatencyBenchEntity {
       )
 }
 
-class LatencyBenchEntity extends Actor with ActorLogging { // FIXME: FIX CASSANDRA AND DO PERSISTENCE BENCH
+class LatencyBenchEntity extends PersistentActor with ActorLogging {
   import LatencyBenchEntity._
 
   log.info(s"Started ${self.path.name}")
   
   var persistentPingCounter = 0
-//  override def persistenceId: String = self.path.name
+  def persistenceId: String = self.path.name
 
-  val started = System.currentTimeMillis()
+  val started = System.nanoTime()
+  lazy val recovered = System.nanoTime()
 
-  override def receive = {// Command: Receive = {
+  // def receive = {
+  def receiveCommand = {
     case msg: PingFirst =>
       // simple roundtrip
       sender() ! PongFirst(msg, started)
```

Comment:
`started` looks wrong, it should be a duration in micros (according to the parameter name)